src/components/About.js

```diff
@@ -1,8 +1,34 @@
-import React from "react";
+import React from 'react';
 
 const About = () => {
   return (
-    <h1> Hello This is About page</h1>
+    <main className="mx-5 my-3">
+      <h2 className="text-uppercase">
```

Comment:
This should be `h1` since it's best practise for each page to have a top level heading. I also commonly see headings as normal case rather than uppercase, so removing `text-uppercase` would be better in my opinion. Uppercase makes it look like "shouting" to some people

Reply:
Done